Feature/footer

cwang04 opened the pull request:
Going through all the branches and writing pull requests.

The website looks good, but we need to change the top bar to a hamburger layout for mobile layout eventually.

## Changes:
- Added desktop top nav bar that has navigation to all of the places on the website.
- Added a footer that has links to connect to social media Instagram, Discord, and TikTok

serrachow committed:
added footer

serrachow committed:
created navbar buttons

angela139 reviewed (commented):
We probably need to change everything based on the Figma

angela139 reviewed (commented):
Testing the discord webhook

angela139 reviewed (commented):
Testing webhook because Colin can't read instructions apparently

angela139 committed:
Merge branch 'main' into feature/footer

angela139 committed:
Added preliminary changes

erhuang623 reviewed (approved):
Nice addition to all the updated links! also nice fix to styling and adding global colors. The accuracy to the figma is pretty spot on. I would say the spacing between the figma and the branch is slightly different but functionality is there! Then again on the figma it has the keebs logo which might be redundant if we already have it in the header. good work!

angela139 committed:
Simplified footer

angela139 committed:
Merge branch 'main' into feature/footer

angela139 committed:
Fixed styling for hero

erhuang623 committed:
Updated Sponsors

Updated the sponsor carousel to include the ones only on our instagram so far. I am not sure which sponsors are guaranteed but I put those for now. Also im not sure whether to put the ones listed as vendor + sponsor because the "logo" would just be an instagram image. If there is a list of sponsors needed on the sponsor page that would be great.

erhuang623 committed:
Updated vendors

Added vendor logos to list of sponsors. Although some images didn't have any text. I'm not sure how the hyperlinks to their websites work since i just took the images from google images.
